.env.local

```diff
@@ -1,2 +1,4 @@
-DB_URL=postgres://postgres:postgres@localhost:5432/postgres
+DB_URL=postgresql://transcription:pBHJSvVuGsMSzGJAgbYu@transcriptiondb.c8wuo4stnbd0.us-west-2.rds.amazonaws.com:5432/postgres
```

Comment:
Isn't this supposed to be ignored in your .gitignore file?

Reply:
not using it, it's a local env for docker-compose. not using it for prod.